app/javascript/toggle_stop_and_route.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener("turbo:load", function () {
```

Comment:
I'd like to see you try turning this into a Stimulus.js controller: https://stimulus.hotwired.dev

I can assure you that Stimulus will _definitely_ feel unnatural at first, but we'll be using it extensively in the near future, and I'd love to see you get comfortable using it :)

Reply:
Of course, this looks interesting.

Reply:
Done !